airgun/views/computeresource.py

```diff
@@ -0,0 +1,105 @@
+from widgetastic.widget import (
+    Checkbox,
+    Select,
+    Text,
+    TextInput,
+    View,
+    ConditionalSwitchableView,
+)
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import FilteredDropdown, ActionsDropdown, SatTable
+
+
+class ComputeResourcesView(BaseLoggedInView, SearchableViewMixin):
+
+    title = Text("//h1[text()='Compute Resources']")
+    new = Text("//a[contains(@href, '/compute_resources/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        """Check if the right page is displayed"""
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class ResourceProviderDetailsView(BaseLoggedInView):
+    name = TextInput(id='compute_resource_name')
+    description = TextInput(id='compute_resource_description')
+    api4 = Checkbox(id='compute_resource_use_v4')
```

Comment:
where did you find that checkbox?
![image](https://user-images.githubusercontent.com/10895065/40610878-bcbd48a6-627c-11e8-970e-b69ae85678d0.png)

Reply:
@lhellebr This is your code. Where did you find it?

Reply:
That's a checkbox valid for RHEV provider in versions supporting RHEV APIv4.

Reply:
So when you select provider RHV you can see checkbox Use APIv4 (experimental)

Reply:
so put that locator in corresponding provider, not on generic level